src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -116,13 +116,14 @@
                 <RowDefinition Height="*" />
                 <RowDefinition Height="Auto" />
             </Grid.RowDefinitions>
-            <ScrollViewer Grid.Row="0">
-                <Frame x:Name="contentFrame" />
-            </ScrollViewer>
+            <Frame x:Name="contentFrame"
+                   Grid.Row="0"/>
+            <!-- Explicitly set the background color on grid to prevent the navigation animation from overflowing it -->
             <Grid Grid.Row="1"
                   Height="100"
                   BorderBrush="{ThemeResource SystemBaseLowColor}"
-                  BorderThickness="0,1,0,0">
+                  BorderThickness="0,1,0,0"
+                  Background="{ThemeResource SystemAltHighColor}">

```

Comment:
This color looks right to me in dark and light themes, but I'm not familiar with which colors are actually supposed to be used. My first attempt `SystemColorWindowColor` turned out not to support themes.

Reply:
Would it also fix the issue if you make the background `Transparent`?

Reply:
It does not

Reply:
Yea I'll be real, I have no idea why some actually work when theme-switched and others don't. I double checked light & dark & HC and this seems to work with them all.